andela/mmdp_site
# MMDP-358 Fix new user inconsistencies

**What does this PR do?**

Placeholder text for New user should be changed to the text “Please enter a valid email address”.

**Description of Task to be completed?**

- change email placeholder to `Please enter a valid email address`.
- change 'save user' caption to `New User`.

**How should this be manually tested?**

1. Fetch all branches - `git fetch --all`.
2. Checkout to this branch - `git checkout bg-fix-new-user-MMDP-358`.
3. Run the application - `yarn dev-server`.
4. Log in and navigate to `Users`. You will notice that the button to create a new user is now labeled `New User`. 
5. Click on the `New User` button to go to the user creation form. The placeholder for email should now be `Please enter a valid email address`. The submit button caption shoul now be `Save user`.

**Any background context you want to provide?**
None

**What are the relevant Jira issues?**

[MMDP-358](https://viisaus.atlassian.net/browse/MMDP-358)

**Screenshots (if appropriate)**
#### Before
##### Create user action button
![screenshot 2019-03-06 at 15 39 40](https://user-images.githubusercontent.com/8180548/53882004-2b3f1480-4026-11e9-83e8-9dc4a5b99937.png)

##### Create user form
![screenshot 2019-03-06 at 15 39 58](https://user-images.githubusercontent.com/8180548/53882010-3003c880-4026-11e9-8ee6-e6c4593c7d03.png)

#### After
##### Create user action button
![screenshot 2019-03-06 at 10 51 14](https://user-images.githubusercontent.com/8180548/53881790-95a38500-4025-11e9-8566-06ff2cb3227b.png)

##### Create user form
![screenshot 2019-03-06 at 15 49 35](https://user-images.githubusercontent.com/8180548/53882498-87eeff00-4027-11e9-998e-522044953e14.png)

**Questions:**
None
